From 585824c33546d1c89236da9350518bde085c9f1a Mon Sep 17 00:00:00 2001
From: Dennis Cagle <d-cagle@codeaurora.org>
Date: Wed, 30 Nov 2016 15:55:19 -0800
Subject: [PATCH] Assign proc_t to proc dir

---
 policy/modules/kernel/files.fc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy/modules/kernel/files.fc b/policy/modules/kernel/files.fc
index c5012e6b4..9938be697 100644
--- a/policy/modules/kernel/files.fc
+++ b/policy/modules/kernel/files.fc
@@ -144,8 +144,8 @@ HOME_ROOT/lost\+found/.*	<<none>>
 #
 # /proc
 #
-/proc			-d	<<none>>
-/proc/.*			<<none>>
+/proc			-d	gen_context(system_u:object_r:proc_t,s0)
+/proc/.*			gen_context(system_u:object_r:proc_t,s0)
 
 #
 # /run
