From 2dd42e3a4330f3f04fd1b1e7e08be48a43233032 Mon Sep 17 00:00:00 2001
From: Dennis Cagle <d-cagle@codeaurora.org>
Date: Wed, 30 Nov 2016 15:55:19 -0800
Subject: [PATCH] Assign proc_t to proc dir

---
 policy/modules/kernel/files.fc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy/modules/kernel/files.fc b/policy/modules/kernel/files.fc
index 677ae96c3..e68f54a32 100644
--- a/policy/modules/kernel/files.fc
+++ b/policy/modules/kernel/files.fc
@@ -145,8 +145,8 @@ HOME_ROOT/lost\+found/.*	<<none>>
 #
 # /proc
 #
-/proc			-d	<<none>>
-/proc/.*			<<none>>
+/proc			-d	gen_context(system_u:object_r:proc_t,s0)
+/proc/.*			gen_context(system_u:object_r:proc_t,s0)
 
 #
 # /run
